frontend/pages/index.tsx

```diff
@@ -92,6 +100,31 @@ export default function Home(): JSX.Element {
           );
         })}
       </div>
+
+      <Flex direction="column" align="center" width="lg">
```

Comment:
I think it would be good to place this in a component like `<HeroSection />` just so that each section is compartmentalized, and if they grow too big we can isolate them into their own files in `Components/`